draft-ietf-ppm-dap.md

```diff
@@ -914,6 +952,45 @@ may occur multiple times for each task, with the Collector obtaining multiple
 aggregate results. For example, imagine tasks where the Collector obtains
 aggregate results once an hour, or every time 10,000,000 reports are uploaded.
 
+~~~ aasvg
+            from aggregation interaction
+                          |
+                          v
+        .-+-----+---------+-------------+-+-----.
+        | |     |                       | |     |
+        v |     |                       v |     |
+Leader batch bucket 1            Helper batch bucket 1
+        | |     |                       | |     |
+        | v     |                       | v     |
+  Leader batch bucket 2            Helper batch bucket 2
+        | |     |                       | |     |
+        | |     v                       | |     v
+    Leader batch bucket m              Helper batch bucket m
+        | |     |                       | |     |
+        | |     |     query chosen      | |     |
+        v v ... v     by Collector      v v ... v
+     .--+-+-----+--.        |        .--+-+-----+--.
+    |  merge Leader |       v       | merge Helper  |
+    | batch buckets |<------+------>| batch buckets |
+     '------+------'                 '------+------'
+            |                               |
+            v                               v
+  Leader aggregate share          Helper aggregate share
+            |                               |
+            '---------------+---------------'
+                            |
```

Comment:
Should this indicate how the shares are encrypted, sent to the collector (via the leader, I believe), decrypted, then unsharded.  That's probably another diagram's worth.

Reply:
> This diagram deliberately does not illustrate how this plays out over a network. 

I didn't pick up on this.  It might be worth making that explicit.

Reply:
It says as much in the first paragraph under "Lifecycle of Protocol Objects":

> The following diagrams illustrate how the various objects in the protocol are
> constructed or transformed into other protocol objects. Oval nodes are verbs or
> actions which process, transform or combine one or more objects into one or more
> other objects. **This diagram does not necessarily illustrate how participants
> communicate. In particular, the processing of aggregation jobs happens in
> distinct, non-colluding parties.**

I'm open to suggestions for making that more clear.